packages/react-component-library/src/components/Modal/Modal.tsx

```diff
@@ -11,6 +11,10 @@ import { useExternalId } from '../../hooks/useExternalId'
 import { useOpenClose } from '../../hooks/useOpenClose'
 
 export interface ModalProps extends ComponentWithClass {
+  /**
+   * Optional override for the accessible name of the Modal.
+   */
+  'aria-label'?: string
```

Comment:
The error for Modal was:

https://dequeuniversity.com/rules/axe/4.4/aria-dialog-name

This was specific to Modals without a title and due to such Modals not having an accessible name.

This was tricky to resolve without any breaking changes, and also taking into account that Dialog is also partly based on Modal.

Personally I'm not sure why title is optional – but this simply makes it possible to explicitly set `aria-label` for a Modal to resolve this without any breaking changes.